am-utils-6.2 - convert AC_HELP_STRING to AS_HELP_STRING

From: Ian Kent <raven@themaw.net>

autoconf 2.71 autoreconf tells us AC_HELP_STRING is obsolete.
Use AS_HELP_STRING instead.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 m4/macros/opt_amu_cflags.m4 |    2 +-
 m4/macros/opt_cppflags.m4   |    2 +-
 m4/macros/opt_debug.m4      |    2 +-
 m4/macros/opt_ldflags.m4    |    2 +-
 m4/macros/opt_libs.m4       |    2 +-
 m4/macros/with_addon.m4     |    2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/m4/macros/opt_amu_cflags.m4 b/m4/macros/opt_amu_cflags.m4
index 1194786b..e94dbc24 100644
--- a/m4/macros/opt_amu_cflags.m4
+++ b/m4/macros/opt_amu_cflags.m4
@@ -4,7 +4,7 @@ dnl NOTE: this is only for final compiltions, not for configure tests)
 AC_DEFUN([AMU_OPT_AMU_CFLAGS],
 [AC_MSG_CHECKING(for additional C option compilation flags)
 AC_ARG_ENABLE(am-cflags,
-AC_HELP_STRING([--enable-am-cflags=ARG],
+AS_HELP_STRING([--enable-am-cflags=ARG],
 		[compile package with ARG additional C flags]),
 [
 if test "$enableval" = "" || test "$enableval" = "yes" || test "$enableval" = "no"; then
diff --git a/m4/macros/opt_cppflags.m4 b/m4/macros/opt_cppflags.m4
index 9cb72362..97197212 100644
--- a/m4/macros/opt_cppflags.m4
+++ b/m4/macros/opt_cppflags.m4
@@ -4,7 +4,7 @@ dnl NOTE: this is for configuration as well as compilations!
 AC_DEFUN([AMU_OPT_CPPFLAGS],
 [AC_MSG_CHECKING(for configuration/compilation (-I) preprocessor flags)
 AC_ARG_ENABLE(cppflags,
-AC_HELP_STRING([--enable-cppflags=ARG],
+AS_HELP_STRING([--enable-cppflags=ARG],
 	[configure/compile with ARG (-I) preprocessor flags]),
 [
 if test "$enableval" = "" || test "$enableval" = "yes" || test "$enableval" = "no"; then
diff --git a/m4/macros/opt_debug.m4 b/m4/macros/opt_debug.m4
index b6f044e9..bca938ac 100644
--- a/m4/macros/opt_debug.m4
+++ b/m4/macros/opt_debug.m4
@@ -4,7 +4,7 @@ dnl and "no" means none.
 AC_DEFUN([AMU_OPT_DEBUG],
 [AC_MSG_CHECKING(for debugging options)
 AC_ARG_ENABLE(debug,
-AC_HELP_STRING([--enable-debug=ARG],[enable debugging (yes/mem/no)]),
+AS_HELP_STRING([--enable-debug=ARG],[enable debugging (yes/mem/no)]),
 [
 if test "$enableval" = yes; then
   AC_MSG_RESULT(yes)
diff --git a/m4/macros/opt_ldflags.m4 b/m4/macros/opt_ldflags.m4
index cf3ee432..51c66601 100644
--- a/m4/macros/opt_ldflags.m4
+++ b/m4/macros/opt_ldflags.m4
@@ -4,7 +4,7 @@ dnl NOTE: this is for configuration as well as compilations!
 AC_DEFUN([AMU_OPT_LDFLAGS],
 [AC_MSG_CHECKING(for configuration/compilation (-L) library flags)
 AC_ARG_ENABLE(ldflags,
-AC_HELP_STRING([--enable-ldflags=ARG],
+AS_HELP_STRING([--enable-ldflags=ARG],
 		[configure/compile with ARG (-L) library flags]),
 [
 if test "$enableval" = "" || test "$enableval" = "yes" || test "$enableval" = "no"; then
diff --git a/m4/macros/opt_libs.m4 b/m4/macros/opt_libs.m4
index e7414860..40d496ba 100644
--- a/m4/macros/opt_libs.m4
+++ b/m4/macros/opt_libs.m4
@@ -4,7 +4,7 @@ dnl NOTE: this is for configuration as well as compilations!
 AC_DEFUN([AMU_OPT_LIBS],
 [AC_MSG_CHECKING(for configuration/compilation (-l) library flags)
 AC_ARG_ENABLE(libs,
-AC_HELP_STRING([--enable-libs=ARG],
+AS_HELP_STRING([--enable-libs=ARG],
 		[configure/compile with ARG (-l) library flags]),
 [
 if test "$enableval" = "" || test "$enableval" = "yes" || test "$enableval" = "no"; then
diff --git a/m4/macros/with_addon.m4 b/m4/macros/with_addon.m4
index d8ea8cfa..c3d8a7fa 100644
--- a/m4/macros/with_addon.m4
+++ b/m4/macros/with_addon.m4
@@ -4,7 +4,7 @@ AC_DEFUN([AMU_WITH_ADDON],
 [AC_MSG_CHECKING([if $1 is wanted])
 ac_upcase=`echo $1|tr 'abcdefghijklmnopqrstuvwxyz' 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'`
 AC_ARG_WITH($1,
- AC_HELP_STRING([--with-$1],
+ AS_HELP_STRING([--with-$1],
 		[enable $2 support (default=yes if found)]
 ),[
 if test "$withval" = "yes"; then
